.gitignore

```diff
@@ -22,3 +22,5 @@ out
 test_assets/
 tmp/
 mobile/repo
+
+.DS_STORE
```

Comment:
This is OS specific. I recommend setting up a global `.gitignore` config on your machine and you will never have to add these per repo. 

Add all the things you want to ignore across all repos on your machine to `~/.gitignore` (like OS-specific files, files specific to the IDEs you use, etc.)  and change `~/.gitconfig` to include:

```
[core]
  excludesfile = ~/.gitignore
```

Reply:
done